Makefile

```diff
@@ -17,4 +17,8 @@ down:
 	- docker-compose -f docker-compose.yml -f docker-compose.local.yml down
 
 test:
+	- rm -rf source/storage/database.sqlite
+	- touch source/storage/database.sqlite
+	- docker exec -it squabble_web_1 php artisan migrate
 	- docker exec -it squabble_web_1 phpunit
+	- vendor/bin/behat
```

Comment:
Similar to above, I would prefer that this is run in the container. Because of the bug with exit codes and Ubuntu and Docker, it would need to be `docker exec squabble_web_1 vendor/bin/behat` (no `-it` flags)